### PR TITLE
64-bit build

### PR DESCRIPTION
**Pull request by SmilyOrg**

### CODE REVIEW

File: loom/common/platform/platformNetworkTests.cpp [hunk 1]
```diff
@@ -32,7 +32,7 @@ SEATEST_TEST(platformNetwork_googlePing)
 {
     loom_net_initialize();
 
-    loom_socketId_t s = loom_net_openTCPSocket("google.com", 80, 1);
+    loom_socketId_t s = loom_net_openTCPSocket("google.si", 80, 1);
```
Comment (bengarney) [inline loom/common/platform/platformNetworkTests.cpp]: Hah.

File: loom/script/compiler/lsJitTypeCompiler.cpp [hunk 2]
```diff
@@ -227,8 +227,14 @@ void JitTypeCompiler::compile(ClassDeclaration *classDeclaration)
...
 
-    compiler.lineNumber = 0;
     compiler.cls        = classDeclaration;
+
+    compiler.lineNumber = 0;
+    compiler.twoSlotFrameInfo = 0;
+    compiler._compile();
+
+    compiler.lineNumber = 0;
+    compiler.twoSlotFrameInfo = 1;
```
Comment (bengarney) [inline loom/script/compiler/lsJitTypeCompiler.cpp]: Codename GEMINI.

File: loom/script/native/lsLuaBridge.h [hunk 3]
```diff
@@ -1531,7 +1531,7 @@ static inline void *const getIdentityKey()
     // or compiler dependent as seems to be a problem which exhibited
     // issues upon a threshold
 
-    return (void *)-0x12345678;
+    return (void *)0x12345678;
```
Comment (bengarney) [inline loom/script/native/lsLuaBridge.h]: Why positive now?

Comment (SmilyOrg) [inline loom/script/native/lsLuaBridge.h]: I'm 90% positive I got compiler errors on some configurations.

File: loom/common/platform/platformNetworkTests.cpp (hunk 1 above)
Comment (SmilyOrg) [inline loom/common/platform/platformNetworkTests.cpp]: When did this happen? :smile:

Comment (themuflon) [inline loom/common/platform/platformNetworkTests.cpp]: Must have forgot it there while debugging the test that was hanging. Change that back to .com :D

**Pull request merged**